keps/sig-scheduling/4815-dra-partitionable-devices/README.md

```diff
@@ -547,107 +806,126 @@ devices:
         int: 3
       uuid:
         string: GPU-662077db-fa3f-0d8f-9502-21ab0ef058a2
+    consumesCapacity:
+    - capacityPool: gpu-3-pool
+      includes:
+      - name: common-gpu-consumption
 ```
 
-As you can see, three mixins are created called "system-attributes",
+As you can see, a capacity pool mixin is created and it defines the
+capacities available for each gpu. This is then referenced in four
+separate capacity pools, each of which maps to a single physical
+GPU.
+Then, three device mixins are created called "system-attributes",
 "common-gpu-attributes", and "common-gpu-capacities", which all get included in
 the definitons of the actual GPU devices themselves, along with their own
-device-specific attributes (and device-specific capacities if there were any).
+device-specific attributes (and device-specific capacities if there were any). 
+A device capacity consumption mixin is also created called "common-gpu-consumption"
+which is referenced from all the devices together with a reference to the
+capacity pool from which the capacities will be drawn.
 
 With this in place, the scheduler can parse these device definitions and
-"flatten" them into something that looks exactly the same as a `Basic` device.
-The scheduler can then allocate such devices using the same algorithm it uses
-for `Basic` devices.
+"flatten" them into a set devices that pull capacity from a set of pools. The
```

Comment:
Can we include something like a kubectl cmd (not sure where it should go) or even a non-default plugin which does the flatten operation and prints the flattened thing?

Maybe we actually want a generic verb for this which fetches related things and prints them all together?  `kubectl flatten gateway foo`  

@kubernetes/sig-cli-leads

Reply:
We have a similar concern over in https://github.com/kubernetes/enhancements/pull/5034: I definitely see a need to improve the CLI to help users make sense of the information that is available because it's not all in one object or even a single type. But not for alpha, and perhaps even as a separate KEP if it's considered non-blocking for graduation of the functionality itself.